conanfile.py

```diff
@@ -23,37 +24,48 @@ class LibtirpcConan(ConanFile):
         "shared": False,
         "fPIC": True,
     }
-    _source_subfolder = "sources"
+    _source_subfolder = "source_subfolder"
+    _autotools = None
 
     def configure(self):
-        if self.settings.compiler in ("gcc", "clang"):
-            del self.settings.compiler.libcxx
+        if self.settings.os != "Linux":
+            raise ConanInvalidConfiguration("libtirpc is only supported for Linux")
+        del self.settings.compiler.libcxx
 
     def source(self):
-        url = "https://sourceforge.net/projects/{name}/files/{name}/{version}/{name}-{version}.tar.bz2".format(
-            name=self.name, version=self.version
+        url = "{homepage}/files/{name}/{version}/{name}-{version}.tar.bz2".format(
+            homepage=self.homepage, name=self.name, version=self.version
         )
         sha256 = "2ca529f02292e10c158562295a1ffd95d2ce8af97820e3534fe1b0e3aec7561d"
         tools.get(url, sha256=sha256)
         os.rename("{}-{}".format(self.name, self.version), self._source_subfolder)
 
+    def _configure_autotools(self):
+        if not self._autotools:
+            conf_args = [
+                "--disable-gssapi",
+                "--enable-ipv6",
+                "--enable-shared" if self.options.shared else "--disable-shared",
+                "--disable-static" if self.options.shared else "--enable-static",
+                "--with-pic" if self.options.fPIC else "--without-pic",
+            ]
+            self._autotools = AutoToolsBuildEnvironment(self)
+            self._autotools.configure(args=conf_args)
+        return self._autotools
+
     def build(self):
-        conf_args = [
-            "--disable-gssapi",
-            "--enable-ipv6",
-            "--enable-shared" if self.options.shared else "--disable-shared",
-            "--disable-static" if self.options.shared else "--enable-static",
-            "--with-pic" if self.options.fPIC else "--without-pic",
-        ]
-        autotools = AutoToolsBuildEnvironment(self)
-        autotools.configure(configure_dir=os.path.join(self.source_folder, self._source_subfolder), args=conf_args)
-        autotools.make()
+        with tools.chdir(self._source_subfolder):
```

Comment:
By changing directory to `self._source_subfolder` instead of using `autogools.configure(configure_dir=...)`, you break the package development flow (because in the development flow the sources are not copied to the build folder).
Having package development flow working is imho very useful while creating/building/troubleshooting a conan recipe.

Reply:
I gonna revert it. thanks

Reply:
When I create a conan recipe, I use the package development flow:
- `conan source -sf src $CONANPATH` to copy the sources
- `conan install -if bin $CONANPATH madebr/testing` to install the dependencies
- `conan build -if bin -bf bin -sf src -pf pkg $CONANPATH` to build the package
- `conan package -if bin -bf bin -sf src -pf pkg $CONANPATH` to create the package

In the build step, the source folder is not copied to the build folder, as it does in `conan export`.

Reply:
This also allows to enable `no_copy_source`.

Reply:
@madebr done!